### DISCUSSION TITLE
Formatted id with leading zeroes

### DISCUSSION THREAD

**Discussion by leonlazic** [Budibase/budibase#13018] [Help]
I would like to create an ID with leading zeroes. I know I need to have the Auto ID column and parse that in a static formula. I am wondering how would that be done easiest? The handlebar helpers don't have a "right" helper like excel or airtable do where I could just have 00000 and replace the digits from right to left to have 00001, 00002, ... 00010, ...

Attached is how I had it setup in Airtable just for better undertstanding.
![Screenshot](https://github.com/Budibase/budibase/assets/26551422/c689023f-1b9f-47da-9420-715615440214)
![Screenshot](https://github.com/Budibase/budibase/assets/26551422/a18d3012-2ab0-4f32-973b-73067a9da1bb)

**Comment by melohagan** [comment]
Hey @leonlazic 

Easiest thing to do would be use a JavaScript formula:

```js
return $("my_auto_id").toString().padStart(6, "00000")
```
